example/todomvc/components/itemview.go

```diff
@@ -81,6 +88,7 @@ func (p *ItemView) Render() *vecty.HTML {
 				"completed": p.Item.Completed,
 				"editing":   p.editing,
 			},
+			prop.ID(p.Item.Title),
```

Comment:
What is this for ?

Reply:
My bad, I left this in from my testing - just made it easier to identify items from devtools, will remove.